### PR TITLE
Teams-Fx Toolkit Migration Nodejs

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5711383934
image

**Review by Prithvi-MSFT — APPROVED** [review]

File: samples/graph-channel-lifecycle/nodejs/teamsapp.local.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      teamsAppId: TEAMS_APP_ID
+
+  - uses: script
+    with:
+      run:
+        # echo "::set-teamsfx-env MICROSOFT_APP_TYPE=SingleTenant";
+        echo "::set-teamsfx-env MICROSOFT_APP_TYPE=MultiTenant";
+        echo "::set-teamsfx-env MICROSOFT_APP_TENANT_ID=${{AAD_APP_TENANT_ID}}";
+
+  - uses: arm/deploy # Deploy given ARM templates parallelly.
```
Comment (formulahendry) [inline samples/graph-channel-lifecycle/nodejs/teamsapp.local.yml:37]: Does this sample has bot? If not, remove bot related things.

Comment (Harikrishnan-MSFT) [reply samples/graph-channel-lifecycle/nodejs/teamsapp.local.yml]: Fixed

File: samples/graph-channel-lifecycle/nodejs/infra/azure.bicep
Comment (formulahendry) [inline samples/graph-channel-lifecycle/nodejs/infra/azure.bicep:1]: Why do we still need infra folder??

File: samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            "command": "debug-start-local-tunnel",
+            "args": {
+                "type": "dev-tunnel",
+                "ports": [
+                    {
+                        "portNumber": 3978,
+                        "protocol": "http",
+                        "access": "public",
+                        "writeToEnvironmentFile": {
+                            "endpoint": "BOT_ENDPOINT", // output tunnel endpoint as BOT_ENDPOINT
```
Comment (formulahendry) [inline samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json:50]: BOT_ENDPOINT?

File: samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                            "domain": "BOT_DOMAIN" // output tunnel domain as BOT_DOMAIN
```
Comment (formulahendry) [inline samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json:51]: BOT_DOMAIN

**Review by formulahendry — CHANGES_REQUESTED** [review]
It is a tab project, please DO NOT use CLI with `-t bot` to do migration

File: samples/graph-channel-lifecycle/nodejs/infra/azure.bicep
Comment (Harikrishnan-MSFT) [reply samples/graph-channel-lifecycle/nodejs/infra/azure.bicep:1]: Fixed

File: samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json]: Fixed

File: samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/graph-channel-lifecycle/nodejs/.vscode/tasks.json]: Fixed

**Review by formulahendry — APPROVED** [review]

**Pull request merged**